sources/packages/web/src/components/aest/students/assessment/ManualReassessment.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-card>
+    <v-container>
+      <TriggerReassessmentModal ref="reassessmentModal" />
+      <body-header title="Trigger Reassessment Manually" class="m-1">
+      </body-header>
+      <content-group class="mt-4 pb-16">
+        <p>
+          Manual system trigger to re-perform assessment using existing
+          application inputs. This should be used as a means of triggering the
+          assessment or other downstream actions (COE requests, eCert requests)
+          without requiring the student to edit and resubmit their application.
+        </p>
+        <check-permission-role :role="Role.AESTManualTriggerReassessment">
+          <template #="{ notAllowed }">
+            <v-btn
+              class="ml-2 float-right"
+              color="primary"
+              prepend-icon="fa:fa fa-refresh"
+              :disabled="notAllowed || openModalButtonDisabled"
+              @click="openTriggerReassessmentModal()"
```

Comment:
Minor and optional. Click can be used like this if there are no arguments `@click="openTriggerReassessmentModal"`